src/viam/sdk/common/client_helper.hpp

```diff
@@ -12,7 +14,22 @@ namespace client_helper_details {
 [[noreturn]] void errorHandlerReturnedUnexpectedly(const ::grpc::Status&) noexcept;
 }  // namespace client_helper_details
 
-template <typename ClientType, typename StubType, typename RequestType, typename ResponseType>
+// Method type for a gRPC call that returns a response message type.
+template <class StubType, class RequestType, class ResponseType>
```

Comment:
Any reason for `class` here rather than `typename` in the template parameter list, and below? I know they mean the same thing, basically, but I think the codebase is pretty consistent in favor of `typename` in that role.

Reply:
oh whoops you're right, no reason just force of habit, I'll switch it to `typename`